guides/common/modules/proc_updating-operating-system.adoc

```diff
@@ -0,0 +1,38 @@
+[id="updating-operating-system_{context}"]
+= Updating the operating system
```

Comment:
title should read `Updating Satellite`   or  `Updating Satellite and operating system`
It is always update of `Satellite` + `OS`

Reply:
The sections above have `Updating Satellite Server` already. I think it makes sense for a separate one to be `Updating the Operating System`, but can we get a capitalization for Operating System so it looks consistent.